ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly.yaml

```diff
@@ -2441,6 +2441,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-cert-rotation
     workflow: baremetal-lab-ipi
+- as: e2e-metal-ovn-two-node-arbiter-disasterrecovery-destructive
```

Comment:
Refer to https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md#job-naming for how the other jobs are named.